spec/lib/task_helpers/imports/custom_buttons_spec.rb

```diff
@@ -5,7 +5,11 @@
   let(:custom_button_set_name)        { 'group1|Vm|' }
   let(:custom_button_set_description) { 'group1' }
   let(:resource_action_ae_namespace)  { 'SYSTEM' }
-  let(:options)                       { {:source => source} }
+  let(:connect_dialog_by_name) { true }
+  let(:options) { {:source => source, :connect_dialog_by_name => connect_dialog_by_name} }
+  let(:button_1) { 'button 1' }
```

Comment:
This variable never changes. Can be removed.

Reply:
right. :button_1 is constant here, defined via let, and used at line 85:
 `btn1 = CustomButton.find_by(:name => button_1)`
Do you want me to use literal instead? like
 `btn1 = CustomButton.find_by(:name => 'button 1')`
Using 'let' to define constant is found as preffered way: see
https://stackoverflow.com/questions/5230070/scope-constants-to-an-rspec-context

Reply:
@lfu @chessbyte  done